website/docs/r/monitor_aad_diagnostic_setting.html.markdown

```diff
@@ -95,7 +95,7 @@ A `enabled_log` block supports the following:
 
 * `category` - (Required) The log category for the Azure Active Directory Diagnostic.
 
-* `retention_policy` - (Required) A `retention_policy` block as defined below.
+* `retention_policy` - (Optional) A `retention_policy` block as defined below.
```

Comment:
The documentation for this field can actually be removed as we don't want anyone to use this going forward

Reply:
updated